taskcluster/fenix_taskgraph/worker_types.py

```diff
@@ -94,3 +94,26 @@ def build_push_apk_payload(config, task, task_def):
             scope_prefix, worker["product"], ":dep" if worker["dep"] else ""
         )
     )
+
+
+@payload_builder(
+    "scriptworker-shipit",
+    schema={
+        Required("upstream-artifacts"): [
```

Comment:
Nit: Do we need `upstream-artifacts` here? I don't see it defined in gecko, for instance https://searchfox.org/mozilla-central/rev/b243debf6235b050b42fd2eb615fdc729636ca6b/taskcluster/taskgraph/transforms/task.py#1212-1220

Reply:
I added upstream-artifacts merely because the multi_dep transforms add it, and I didn't think it was worth removing it.... running the full graph I didn't see them used. Am I wrong in that assumption for here?

Reply:
Your approach makes sense to me 🙂 I think we should leave a comment that explains why it's here while not used. This way, we won't confuse future readers.